### PR TITLE
[WIP] Start getting an interface for MySQL

### PR DESCRIPTION
**Pull request by rudehn**
Just starting to flesh out some code

### CODE REVIEW

**Comment by PlasmaPower** [comment]
Mark as WIP? or do you think it's done enough to merge as a base?

**Comment by hongaar** [comment]
Would sqlite be an option? Waaay less overhead and waaay more better unless we're going to build the next social network...?

**Comment by rudehn** [comment]
I would prefer it, but mysql was already present in the Dockerfile. I'm not sure if it's being used anywhere in production though.

**Review by syvb — APPROVED** [review]

**Review by syvb — APPROVED** [review]

File: db/db.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import pymysql
+import logging
+
+
```
Comment (andrewda) [inline db/db.py:4]: Remove extra new line (example from elsewhere in the code: https://github.com/chaosbot/Chaos/blob/9dabaf3349f64c9b7a4b9d32e3a1ccb7afa8122e/cron/poll_issue_close_stale.py#L7)

**Review by andrewda — CHANGES_REQUESTED** [review]
Looks good, just one tiny change

**Comment by rudehn** [comment]
See #456

**Comment by hongaar** [comment]
Let's keep it though, who knows we'll be building the next social network anyway: #461

**Pull request closed without merging**